gf-guide/diacritics.md

```diff
@@ -17,70 +17,70 @@ Many diacritics are separated from the base letter and can be placed above, belo
 
-Anchors are commonly represented as red rhombus in the glyph view of the source file and are identified with a name. The name word should be shared among base and mark, but with a preceding underscore in the latter. E.g. `top` in the base letter and `_top` in the mark
-Anchors' name schema is crucial for the positioning to work as expected, for example if the underscore is ommited in the mark it would not be attached to the base letter, so you must pay special care and attention to them.  
+Anchors are commonly represented as a red rhombus in the glyph view of the source file and are identified with a name. The name part should be shared among the base glyph and the mark glyph, but in the mark glyph there should be a preceding underscore. For example, there should be a `top` anchor in the base glyph and a corresponding `_top` anchor in the mark.
+This name schema is crucial for the positioning to work as expected - for example if the underscore is omitted in the mark glyph, it would not be attached to the base letter - so you must pay special care and attention to them.  
 
 <!-- Include information about stacked diacritics -->
 
 
-**Conditions for combining marks:**
+**Requirements for combining marks:**
 
-- **They must have a zero width value once in the font file `=0`**. During the designg process, you could use a 0 sidebearing value to facilitate the acces to the glyph while working on the source file. But once the fon binaries are complied Glyphs or Fontmake will remove the width value to coply with the requirement that they must be zero value width glyphs, hense **nonspacing**. 
-- Combining marks name should use the *comb* sufix, e.g. `acutecomb`, `gravecomb`. 
-- The outlines desing should follow the design considerations detailed above.
-- They must include anchors named as explained above. To better administrate the positions to language requirements, like stacked diacritics in Vietnamese, it is useful to use custom names for them, e.g. `top_viet` and `_top_viet`, accordingly.
-- The amount of combining marks and precomposed glyphs required will be determine to the language support your font is intended to. For Google Fonts see the [Glyphsets](https://googlefonts.github.io/gf-guide/requirements.html#glyphsets) definition.
-- They must have the right Unicode codepoint in your source file to work properly. See the [Combining diacritical marks](https://unicode.org/charts/PDF/U0300.pdf) Unicode chart, that ranges from 0300 to 036F codepoints.
-- They belong to the mark-nonspacing category in the Glyphs Definiton of the `GDEF` table.
+- **They must have a zero width value once in the font binary `=0`**. During the design process, you could use a 0 sidebearing value to facilitate access to the glyph while working on the source file. But once the font binaries are complied, Glyphs or Fontmake will remove the width value to comply with the requirement that they must be zero value width glyphs (hence the name **nonspacing**).
```

Comment:
instead of "you could use a 0 sidebearing value etc." i would say "While working on the source files, it is recommended to use a positive width with positive sidebearings. Indeed, if the glyphs has anchors and the correct name, the tool generating the font will take care of changing the width to 0".

Reply:
complied -> compiled

Reply:
Agreed